starlark/value.go

```diff
@@ -711,6 +712,21 @@ func (fn *Function) Param(i int) (string, syntax.Position) {
 	id := fn.funcode.Locals[i]
 	return id.Name, id.Pos
 }
+
+// ParamDefault returns the default value the ith optional parameter, where
```

Comment:
Let's say: "ParamDefault returns the default value of the specified parameter (0 <= i < NumParams()), or nil if the parameter is not optional."

(and update the implementation to correspond)

Reply:
Done.